src/ResourceManager/Kubernetes/Commands.Kubernetes/Commands.Kubernetes.csproj

```diff
@@ -0,0 +1,151 @@
+    <Compile Include="Generated\Models\ContainerServiceSshPublicKey.cs" />
+    <Compile Include="Generated\Models\ContainerServiceStorageProfileTypes.cs" />
+    <Compile Include="Generated\Models\ContainerServiceVMDiagnostics.cs" />
+    <Compile Include="Generated\Models\ContainerServiceVMSizeTypes.cs" />
+    <Compile Include="Generated\Models\ContainerServiceWindowsProfile.cs" />
+    <Compile Include="Generated\Models\KeyVaultSecretRef.cs" />
+    <Compile Include="Generated\Models\ManagedCluster.cs" />
+    <Compile Include="Generated\Models\ManagedClusterAccessProfile.cs" />
+    <Compile Include="Generated\Models\ManagedClusterPoolUpgradeProfile.cs" />
+    <Compile Include="Generated\Models\ManagedClusterUpgradeProfile.cs" />
+    <Compile Include="Generated\Models\OrchestratorProfile.cs" />
+    <Compile Include="Generated\Models\OSType.cs" />
+    <Compile Include="Generated\Models\Page.cs" />
+    <Compile Include="Generated\Models\Resource.cs" />
+    <Compile Include="Models\AcsServicePrincipal.cs" />
+    <Compile Include="Models\Mapper.cs" />
+    <Compile Include="Models\PSContainerServiceAgentPoolProfile.cs" />
+    <Compile Include="Models\PSContainerServiceLinuxProfile.cs" />
+    <Compile Include="Models\PSContainerServiceServicePrincipalProfile.cs" />
+    <Compile Include="Models\PSContainerServiceSshConfiguration.cs" />
+    <Compile Include="Models\PSContainerServiceSshPublicKey.cs" />
+    <Compile Include="Models\PSKeyVaultSecretRef.cs" />
+    <Compile Include="Models\PSKubernetesCluster.cs" />
+    <Compile Include="Models\PSResource.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Properties\" />
+  </ItemGroup>
+  <ItemGroup />
+  <ItemGroup>
+    <Reference Include="AutoMapper">
+      <HintPath>..\..\..\packages\AutoMapper.6.2.2\lib\net45\AutoMapper.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="System.Management.Automation, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\System.Management.Automation_PowerShell_3.0.6.3.9600.17400\lib\net40\System.Management.Automation.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="YamlDotNet, Version=4.2.2.0, Culture=neutral, PublicKeyToken=ec19458f3c15af5e, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\YamlDotNet.Signed.4.2.2\lib\net35\YamlDotNet.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\Common\Commands.Common.Authentication.Abstractions\Commands.Common.Authentication.Abstractions.csproj">
+      <Project>{70527617-7598-4aef-b5bd-db9186b8184b}</Project>
+      <Name>Commands.Common.Authentication.Abstractions</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\..\Common\Commands.Common.Authorization\Commands.Common.Authorization.csproj">
```

Comment:
SHould not take this reference - Commands.Common.Authentication is meant to contain realizations of the Abstractions types, no cmdlet assembly other than profile should take a dependency on thsi assembly